packages/zowe-explorer/src/ZoweExplorerExtender.ts

```diff
@@ -44,6 +44,68 @@ import { getProfileInfo, getProfile } from "./utils/ProfilesUtils";
 export class ZoweExplorerExtender implements ZoweExplorerApi.IApiExplorerExtender, ZoweExplorerTreeApi {
     public static ZoweExplorerExtenderInst: ZoweExplorerExtender;
 
+    /**
+     * Shows an error when the Zowe configs are improperly configured, as well as
+     * an option to show the config.
+     *
+     * @param errorDetails Details of the error (to be parsed for config name and path)
+     */
+    public static async showZoweConfigError(errorDetails: string) {
+        vscode.window
+            .showErrorMessage(
+                localize(
+                    "initialize.profiles.error",
+                    'Error encountered when loading your Zowe config. Click "Show Config" for more details.'
+                ),
+                "Show Config"
+            )
+            .then((selection) => {
+                if (selection !== "Show Config") {
+                    return;
+                }
+
+                // Parse the v2 config path if it exists; otherwise look for a v1 config path
+                const configMatch = errorDetails.includes("Error parsing JSON in the file")
+                    ? errorDetails.match(/Error parsing JSON in the file \'(.+?)\'/)
+                    : errorDetails.match(/Error reading profile file \(\"(.+?)\"\)/);
+
+                let configPath = configMatch != null ? configMatch[1] : null;
+                if (configPath == null) {
+                    // If unable to parse config path from error message,
+                    // try to build a v2 config path from zowe dir
+                    configPath = this.getConfigLocation(getZoweDir());
+
+                    // If configPath is still null, we can't find a v2 config - return.
+                    if (configPath == null) {
+                        return;
+                    }
+                }
+                vscode.window.showTextDocument(vscode.Uri.file(configPath));
+            });
+    }
+
+    /**
+     * Returns the location of a Zowe config (if it exists) relative to rootPath.
```

Comment:
@traeok I see this check for config in respect to the zowe home dir.  should we also include a check for config at the project level?

Reply:
For `showZoweConfigError`, in most cases the directory is read from the error message. 

I just pushed an update to this - in edge cases where a directory cannot be parsed, I've added a check within that function to see if the error comes from a global Zowe config. Otherwise, it will assume that the culprit is within a project-level config and will check in the location for the active workspace within VS Code.